docs/source/checks/nlp/data_integrity/plot_conflicting_labels.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__conflicting_labels:
+
+Conflicting Labels
+******************
+
+This notebook provides an overview for using and understanding the Conflicting Labels check:
+
+**Structure:**
+
+* `Why check for conflicting labels? <#why-check-for-conflicting-labels>`__
```

Comment:
just a note for future - I think that if you put only the link directly it will take the exact text of the header, so no need to copy paste it

e.g. this link would probably be rendered the same if it would be:
* \`#why-check-for-conflicting-labels\`__

(yes, without the <>)